packages/server/src/instances/service.ts

```diff
@@ -86,6 +94,16 @@ export class InstanceService extends Service {
     await this.invalidator.setup(this.cache, this.failures)
   }
 
+  async destroy() {
+    const keys = this.cache.keys()
+
+    for (const conduitId of keys) {
+      this.cache.del(conduitId)
+    }
+
+    this.cache.prune()
```

Comment:
Why prune after having deleted all keys?

Reply:
LRU cache can have lazy behavior so here I assure that destroy() gets call on all conduits (otherwise the LRU cache might wait for the next call to `.get` before actually calling destroy.